cms/djangoapps/contentstore/xblock_services/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""
+Xblock services that contain the business logic for xblock views.
+"""
+from .create_xblock import *
+from .xblock_helpers import *
+from .xblock_service import *
```

Comment:
In the context of XBlocks, "services" is already occupied by the idea of [XBlock runtime services](https://github.com/openedx/edx-platform/blob/master/xmodule/services.py). These services, which individual XBlocks request using [@wants and @needs](https://github.com/openedx/XBlock/blob/30d2531eace1bd0b36ed0119eb63ce121d5bd6f0/xblock/mixins.py#L121-L137), allow XBlocks to interface with the outside world without being tied to a particular runtime (eg, LMS, CMS, or xblock-sdk). 

Could you rename this package to avoid confusion?

cc @ormsbee

Reply:
xblock_view_services?